### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by DannieMo**
This is Milestone 1 of the "To-do list" project. This app helps to organize your day by simply listing the things that you need to do and allows you to mark them as complete.

> Task Completed

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<i class="fa-sharp fa-solid fa-trash icon__size"></i>
+</li>
+<hr>
+`,
+  );
+};
+tasks.innerHTML = listArray
+  .reverse()
+  .map((task) => displayTasks(task))
+  .join('');
```
Comment (muneebulrehman) [inline src/index.js]: The tasks you have added in `listArray` are not being rendered on the UI. Kindly fix this issue. Here is the screenshot of the rendered UI. ⬇️
image

File: README.md [hunk 2]
```diff
@@ -1,2 +1,127 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSED) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (muneebulrehman) [inline README.md]: Kindly make sure you have added the section `Future Features`, which includes 1-3 potential features to be added. It is a part of the general requirements for every project you will be working on here at Microverse.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @DannieMo , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add the `Lighthouse` linters and `Webhint` linters to your project as well. These linter checks are missing from your project. You can find the linter config file [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @DannieMo 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**